lib/model/message_list.dart

```diff
@@ -155,7 +155,9 @@ mixin _MessageSequence {
         }
       case MessageListRecipientHeaderItem(:var message):
       case MessageListDateSeparatorItem(:var message):
-        return (message.id <= messageId) ? -1 : 1;
+        // TODO(#1441): test
+        if (message.id == null)                  return 1;
```

Comment:
nit:
```suggestion
        if (message.id == null)                  return 1;  // TODO(#1441): test
```

That makes it clear the comment applies to that line and not the whole case. (I believe we already have a test covering the existing line below; if not, that's independent of #1441.)